libraries/WiFi/examples/WiFiClient/WiFiClient.ino

```diff
@@ -1,29 +1,46 @@
 /*
- *  This sketch sends data via HTTP GET requests to data.sparkfun.com service.
- *
- *  You need to get streamId and privateKey at data.sparkfun.com and paste them
- *  below. Or just customize this script to talk to other HTTP servers.
- *
+    Go to thingspeak.com and create an account if you don't have one already.
```

Comment:
I think moving this part to a README file makes more sense.
https://github.com/espressif/arduino-esp32/blob/master/docs/EXAMPLE_README_TEMPLATE.md
You can use this as a reference.

Reply:
I agree, thank you.